x11-libs/pango/patches/pango-1.50.3.patchset

```diff
@@ -0,0 +1,225 @@
+From be6bdcc2b87f333a5bb1225c89660779fabc2e45 Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
```

Comment:
Could you keep the credits to @korli for the first patch and add your changes on top of it?

Reply:
Fixed.